ci/test.yml

```diff
@@ -308,18 +308,23 @@ connect-popup legacy npm package:
     refs:
       - schedules
 
-.e2e connect-popup-webextension:
+.e2e connect-explorer-webextension:
   stage: integration testing
   retry: 1
   variables:
     COMPOSE_PROJECT_NAME: $CI_JOB_ID
     COMPOSE_FILE: ./docker/docker-compose.connect-popup-ci.yml
-    URL: ${DEV_SERVER_URL}/connect/${CI_COMMIT_REF_NAME}/
-    TEST_FILE: $TEST_FILE
+    # TODO: for some reason current dev server url does not load properly.
+    # TREZOR_CONNECT_SRC: ${DEV_SERVER_URL}/connect/${CI_COMMIT_REF_NAME}/
+    TREZOR_CONNECT_SRC: https://connect.trezor.io/9/
+    URL: https://connect.trezor.io/9/
```

Comment:
shoudln't this get current branch? line 318?

Reply:
Yes, you are right, we had issue with deployments to dev sldev so I changed it, but now the issue is solved and using ` ${DEV_SERVER_URL}/connect/${CI_COMMIT_REF_NAME}/` works and tests are passing.